server/lib/util.py

```diff
@@ -639,3 +640,20 @@ def _get_highest_coverage_date(observation_entity_counts_by_date,
   } for obs in observation_dates]
   best_coverage = max(date_counts, key=lambda date_count: date_count['count'])
   return best_coverage['date']
+
+
+def get_vertex_ai_models():
+  import shared.model.loader as model_loader
```

Comment:
Why import here?

Reply:
originally in __init__.py, this import was inside the `if os.environ.get('ENABLE_EVAL_TOOL') == 'true'`so was just keeping that consistent

Reply:
oh lol I forget why I did that (maybe to not import aiplatform unnecessarily?). but since this is going to be loaded commonly now, maybe ok putting it on top.

Reply:
done